contracts/helpers/WETHOmnibridgeRouter.sol

```diff
@@ -0,0 +1,54 @@
+pragma solidity 0.7.5;
+
+import "../interfaces/IOmnibridge.sol";
+import "../interfaces/IWETH.sol";
+import "../libraries/AddressHelper.sol";
+import "../upgradeable_contracts/modules/OwnableModule.sol";
+import "../upgradeable_contracts/Claimable.sol";
+
+contract WETHOmnibridgeRouter is OwnableModule, Claimable {
+    IOmnibridge public immutable bridge;
+    // solhint-disable-next-line var-name-mixedcase
+    IWETH public immutable WETH;
+
+    constructor(
+        IOmnibridge _bridge,
+        IWETH _weth,
+        address _owner
+    ) OwnableModule(_owner) {
+        bridge = _bridge;
+        WETH = _weth;
+        _weth.approve(address(_bridge), uint256(-1));
+    }
+
+    function wrapAndRelayTokens(address _receiver) external payable {
```

Comment:
may we have simplified version of this call where `msg.sender` is the recipient of tokens on another side of the bridge?